modules/cli/src/main/scala/coursier/cli/bootstrap/GraalvmOptions.scala

```diff
@@ -0,0 +1,29 @@
+package coursier.cli.bootstrap
+
+import caseapp.{ExtraName => Short, HelpMessage => Help, ValueDescription => Value, _}
+
+// format: off
+final case class GraalvmOptions(
+  @Group("Graalvm")
+  @Hidden
+  @Help("Generate a GraalVM native image")
+    nativeImage: Option[Boolean] = None,
```

Comment:
Ok to hide this one, as I'm not sure how well it works, but ultimately, it could be made visible by default.

Reply:
I keep it hidden and add a comment